src/app/(nav-feature)/calendar/page.tsx

```diff
@@ -0,0 +1,6 @@
+"use client";
```

Comment:
Not sure if it's a good idea to declare the whole page as Client component since it's basically a route on the server. I think we should use "use client" for components that need interractions or hook calls. What do you think ?

Reply:
I think 'use client' will be necessary for the translation

Reply:
Next-intl works also on server side

Reply:
We can still remove use client by default and add it if we need it